src/components/PlayerNotesModal/PlayerNotesModal.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2012-2021  Online-Go.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from "react";
+import {_} from 'translate';
+import {Modal, openModal} from "Modal";
+import { get, set } from "remote_storage";
```

Comment:
To avoid naming collisions:

import * as remote_storage from "remote_storage";

See discussion at https://github.com/online-go/online-go.com/pull/1448/files/04f8b723374c9c4f7d8edc572b15ffab6735a77f#r612600953

Reply:
I ended up doing it that way in #1451 in data.ts, for obvious reasons :)

I agree it belongs here (and uniformly).